packages/typespec-vscode/src/extension.ts

```diff
@@ -176,6 +176,8 @@ async function launchLanguageClient(context: ExtensionContext) {
     documentSelector: [
       { scheme: "file", language: "typespec" },
       { scheme: "untitled", language: "typespec" },
+      { scheme: "file", language: "yaml", pattern: "**/tspconfig.yaml" },
+      { scheme: "untitled", language: "yaml", pattern: "**/tspconfig.yaml" },
```

Comment:
```suggestion
      { scheme: "untitled", language: "yaml", pattern: "**/tspconfig.yaml" },
```
is the pattern here doing anything for untitled? should there be a sub yaml language for this instead?

Reply:
good catch. I think the utitled line is not needed. will remove it after confirm

Reply:
line removed.